packages/wonder-blocks-modal/components/modal-content.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View} from "wonder-blocks-core";
+
+export default class ModalContent extends React.Component<{
+    children: React.Node,
+    style?: any,
+}> {
+    render() {
+        const {style, children} = this.props;
+        return <View style={[styles.content, style]}>{children}</View>;
+    }
+}
+
+const styles = StyleSheet.create({
+    content: {
+        flex: "1",
+        padding: 64,
+        overflow: "auto",
```

Comment:
I think we should have some way to easily override this styling.  One of the examples in the modals Zeplin has no left/right padding using the standard modal with preview.

Reply:
Maybe we can have a prop on modals that toggles padding within content panes and by default it's on.

Reply:
Oh, to be clear that's exactly what I'm doing for the preview area (that I already implemented). I pass in a ModalContent and override the `style` to reset the padding to be what I want. I feel like every part of the system can be overridden right now so it's quite flexible!